packages/example/ios/Podfile.lock

```diff
@@ -101,13 +101,16 @@ PODS:
   - GoogleUtilities/UserDefaults (7.11.5):
     - GoogleUtilities/Logger
   - Leanplum-iOS-SDK (4.0.0)
+  - MetricsReporter (1.0.0):
```

Comment:
do we also add metrics in flutter?

Reply:
no we haven't added support for it in the Flutter SDK, but the `MetricsReporter` pod got downloaded, as it was a dependency of the `Rudder` iOS pod